clients/cloud/kotlin/src/main/kotlin/io/confluent/examples/clients/cloud/util/Utils.kt

```diff
@@ -0,0 +1,16 @@
+package io.confluent.examples.clients.cloud.util
+
+import java.io.FileInputStream
+import java.io.IOException
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.util.*
+
+fun loadConfig(configFile: String): Properties {
+  if (!Files.exists(Paths.get(configFile))) {
+    throw IOException("$configFile not found.")
```

Comment:
IllegalArgument?

Reply:
?

Reply:
Is it correct to throw IOException here when there is everything fine with IO?